### PR TITLE
Add theme provider and main page

### PR DESCRIPTION
**Pull request by tinashechiraya**
This pr is linked to ticket:

- https://github.com/kartoza/africa_rangeland_watch/issues/22
- https://github.com/kartoza/africa_rangeland_watch/issues/14

homepage
user_menu
responsive_home_page
footer_responsiveness
footer

### CODE REVIEW

File: django_project/frontend/src/App.tsx [hunk 1]
```diff
@@ -1,28 +1,17 @@
...
+      <Router>
+        <Routes />
+      </Router>
+    </ChakraProvider>
+  );
+}
 
-import './styles/index.scss';
-
-Sentry.init({
```
Comment (dimasciput) [inline django_project/frontend/src/App.tsx:11]: Please don't remove this

File: django_project/frontend/src/App.tsx [hunk 2]
```diff
@@ -1,28 +1,17 @@
...
-    dsn: (window as any).sentryDsn,
-    tunnel: '/sentry-proxy/',
-    tracesSampleRate: 0.5
-})
-
-const rootElement = document.getElementById('app')!
-const root = createRoot(rootElement);
-root.render(
```
Comment (dimasciput) [inline django_project/frontend/src/App.tsx:19]: Please don't remove this

File: django_project/frontend/package.json [hunk 3]
```diff
@@ -3,18 +3,31 @@
...
+    "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "dotenv": "^16.0.1",
+    "framer-motion": "^10.16.5",
     "mini-css-extract-plugin": "^2.9.2",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^18.3.1",
+    "react-datepicker": "^4.5.0",
```
Comment (dimasciput) [inline django_project/frontend/package.json:21]: Do you need this now?

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/src/App.tsx (hunk 2 above)
Comment (tinashechiraya) [reply django_project/frontend/src/App.tsx:19]: @dimasciput this is in index.tsx

**Pull request merged**